js/main.js

```diff
@@ -0,0 +1,119 @@
+'use strict';
+
+var getRandomNumberInRange = function (min, max) {
+  var rand = min + Math.random() * (max + 1 - min);
+  return Math.floor(rand);
+};
+
+var allNames = [
+  `Дима`,
+  `Андрей`,
+  `Майрам`,
+  `Лена`,
+  `Иван`,
+  `Герман`,
+  `Магомед`,
+];
+
+var allMessages = [
+  `Всё отлично!`,
+  `В целом всё неплохо. Но не всё.`,
+  `Когда вы делаете фотографию, хорошо бы убирать палец из кадра. В конце концов это просто непрофессионально.`,
+  `Моя бабушка случайно чихнула с фотоаппаратом в руках и у неё получилась фотография лучше.`,
+  `Я поскользнулся на банановой кожуре и уронил фотоаппарат на кота и у меня получилась фотография лучше.`,
+  `Лица у людей на фотке перекошены, как будто их избивают. Как можно было поймать такой неудачный момент?!`,
+];
+
+
+/* ++++++++++ ++++++++++ ++++++++++ ++++++++++ ++++++++++++++++++++ ++++++++++ */
+// Создаю массив из фотографий
+var gatAllPhotos = function (photoQuantity) {
+  var allPhotos = [];
+
+  for (var i = 1; i < photoQuantity; i++) {
```

Comment:
цикл должен идти до photoQuantity включительно, иначе будет на один элемент меньше чем нужно
```suggestion
  for (var i = 1; i <= photoQuantity; i++) {
```

Reply:
Добавил =